azkaban-common/src/main/java/azkaban/scheduler/ScheduleManager.java

```diff
@@ -102,6 +105,37 @@ public void shutdown() {
     return new ArrayList<>(this.scheduleIDMap.values());
   }
 
+  /**
+   * Retrieves a copy of the list of schedules with filters.
+   */
+  public synchronized List<Schedule> getSchedules(String projectContain,
```

Comment:
please apply save plugin to auto-format your change. Check contributor guide for details.